### PR TITLE
2 new FAQ entries

### LINKED ISSUE #22: The used app permissions should be explained in the FAQs

### PR DESCRIPTION
**Pull request by missytake**
Added two answers to the FAQ, one based on whether we store E-Mail account passwords, the other one about the app's permissions on Android.

closes #22

### CODE REVIEW

**Review by hpk42 — APPROVED** [review]
lgtm but good if @cyberta or @r10s also take a brief look i guess.

File: en/help.md [hunk 1]
```diff
@@ -59,6 +59,47 @@ Other messages do not appear automatically, and are to be found in the main menu
...
+program needs the password so you can use it to send mails. Of course, the
+password is stored only on your device. The password is only transmitted to
+your E-Mail provider (when you login), which has access to your mails anyway. 
+
+If you use an E-Mail provider with OAuth2 support like gmail.com or yandex.ru,
+there is no need to store your password on the device. In this case, only an
+access token is used.
+
+As Delta Chat is Open Source, you can check the [Source
+Code](https://github.com/deltachat/deltachat-core-rust/blob/master/src/login_param.rs)
```
Comment (r10s) [inline en/help.md:74]: nice!

**Review by r10s — APPROVED** [review]
yip, lgtm

**Comment by missytake** [comment]
Thanks :)

File: en/help.md (hunk 1 above)
Comment (missytake) [reply en/help.md:74]: I have to admit It's a bit of a guess whether that's really the only code which handles the passwords...

File: en/help.md [hunk 2]
```diff
@@ -59,6 +59,47 @@ Other messages do not appear automatically, and are to be found in the main menu
...
+  - read the contents of your SD card: to share files with your contacts
+- Other app capabilities
+  - change your audio settings: so you can choose ring tones and volume for notifications and audio messages
+  - run at startup: so you don't have to start Delta Chat manually
+  - control vibration: for notifications
+  - view network connections: to connect to your E-Mail provider
+  - prevent phone from sleeping: so you can easier copy the security code during the Autocrypt Setup Message
+  - have full network access: to connect to your E-Mail provider
+  - view Wi-Fi connections: to connect to your E-Mail provider
+  - set wallpaper
```
Comment (cyBerta) [inline en/help.md:100]: so set wallpaper needs to be removed from that list now after the merge of https://github.com/deltachat/deltachat-android/pull/1142

**Pull request merged**